### PR TITLE
Doc conversion

### PR DESCRIPTION
**Pull request by outoftardis**
# Description

Converted docs from classical Developer Guide: 
- quality metrics, 
- normalization, 
- naive bayes, 
- sum of functions, 
- pca transform, 
- quantiles, 
- qr decomposition, 
- cholesky decomposition, 
- correlation distance, 
- cosine distance, 
- low moments order, 
- adagrad, 
- distributions,
- svd, 
- sorting, 
- covariance,
- outlier detection,
- usage model for recommendation systems,
- engines, 
- adaboost, 
- logitboost, 
- brownboost

Updated copyrights for rst files (either 2020 or 2019-2020 depending on when the rst file was created)

### CODE REVIEW

File: docs/source/dev_guide/algorithms/normalization/min-max.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/normalization/min-max.rst:2]: 2020 only?

File: docs/source/dev_guide/algorithms/normalization/z-score.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/normalization/z-score.rst:2]: 2020 only?

File: docs/source/dev_guide/algorithms/qr/without-pivoting/computation-batch-online.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/qr/without-pivoting/computation-batch-online.rst:2]: 2020 only?

File: docs/source/dev_guide/algorithms/quality_metrics/default/for-binary-classification.rst [hunk 4]
```diff
@@ -0,0 +1,183 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/quality_metrics/default/for-binary-classification.rst:2]: 2020 only?

File: docs/source/dev_guide/algorithms/quality_metrics/default/for-linear-regression.rst [hunk 5]
```diff
@@ -0,0 +1,331 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/quality_metrics/default/for-linear-regression.rst:2]: 2020 only?

File: docs/source/dev_guide/algorithms/quality_metrics/default/for-multi-class-classification.rst [hunk 6]
```diff
@@ -0,0 +1,235 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/quality_metrics/default/for-multi-class-classification.rst:2]: 2020 only?

File: docs/source/dev_guide/algorithms/quality_metrics/default/for-pca.rst [hunk 7]
```diff
@@ -0,0 +1,152 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
```
Comment (averbukh) [inline docs/source/dev_guide/algorithms/quality_metrics/default/for-pca.rst:2]: 2020 only?

**Comment by outoftardis** [comment]
> The only comment is to use year 2020 only for new files, because it's legal requirement.

Even when the content is not new?

**Comment by outoftardis** [comment]
@averbukh I've fixed the copyrights, please take a look and say if it's ok to merge

**Pull request merged**